packages/block-editor/src/components/text-transform-control/stories/index.story.js

```diff
@@ -1,33 +1,59 @@
-/**
- * WordPress dependencies
- */
-import { useState } from '@wordpress/element';
-
 /**
  * Internal dependencies
  */
 import TextTransformControl from '../';
 
-export default {
+const meta = {
 	title: 'BlockEditor/TextTransformControl',
 	component: TextTransformControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Control to facilitate text transformation selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		onChange: {
+			action: 'onChange',
+			control: {
+				type: null,
+			},
+			description: 'Handles change in text transform selection.',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
+		className: {
+			control: { type: 'text' },
+			description: 'Class name to add to the control.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			options: [ 'none', 'uppercase', 'lowercase', 'capitalize' ],
+			control: { type: 'radio' },
+			description: 'Currently selected text transform.',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'none' },
+			},
+		},
 	},
 };
 
-const Template = ( { onChange, ...args } ) => {
-	const [ value, setValue ] = useState();
-	return (
-		<TextTransformControl
-			{ ...args }
-			onChange={ ( ...changeArgs ) => {
-				onChange( ...changeArgs );
-				setValue( ...changeArgs );
-			} }
-			value={ value }
-		/>
-	);
-};
+export default meta;
 
-export const Default = Template.bind( {} );
+export const Default = {
+	args: {
+		value: 'none',
+	},
+	render: function Template( props ) {
+		return <TextTransformControl { ...props } />;
+	},
+};
```

Comment:
This implementation certainly works, but it might be better to use [this previous code](https://github.com/WordPress/gutenberg/pull/67365/files#diff-2b912337cbad0d4b91c3c0bae92eec7676c07504dac1db0971fd8e628a74395eL19-L31) as it is to align with the implementation of other stories.

Reply:
I have reverted to the old code to ensure consistency.